doc/nrf/doc_styleguide.rst

```diff
@@ -57,6 +57,16 @@ It is also possible to define more than one default link text for a link, which
    .. _`Link text one`:
    .. _`Link text two`: http://..
 
```

Comment:
See line 26 - this section should either go up there, or the information about Doxygen should be moved down into the Breathe section.